docs/API/Users_Guide/Data_Analysis.md

```diff
@@ -150,6 +153,379 @@ gene_start      gene_end        symbol  id
+    },
+    "genes": {
+        "details": [
+            {
+                "gene_id": "ENSG00000141510",
+                "has_gene_expression_values": true
+            },
+            {
+                "gene_id": "ENSG00000181143",
+                "has_gene_expression_values": true
+            }
+        ],
+        "with_gene_expression_count": 2,
+        "without_gene_expression_count": 0
+    }
+}
+```
+
+### Gene Expression Values Endpoint
+
+Get gene expression values for the given cases and genes. The response is a tsv containing the expression values for genes to cases.
+
+__Example 1__: A user wants to get expression values using case set id and gene set id.
+
+```Filter
+{
+  "case_set_id": "gOq-6bhPHBg7b-f7dy2nPw",
+  "gene_set_id": "mhiW6Y8PFLqNo7u9qXCtNA",
+  "tsv_units": "uqfpkm",
+  "format": "tsv"
+}
+```
+
+```Shell
+curl -X 'POST' \
+  'https://api.gdc.cancer.gov/gene_expression/values' \
+  -H 'accept: text/tab-separated-values' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "case_set_id": "gOq-6bhPHBg7b-f7dy2nPw",
+  "gene_set_id": "mhiW6Y8PFLqNo7u9qXCtNA",
+  "tsv_units": "uqfpkm",
+  "format": "tsv"
+}'
+```
+
+```Response
+gene_id	e3b32485-b204-43a7-93a5-601408fcdf96
+ENSG00000141510	6.93200
+ENSG00000181143	10.77660
+```
+
+__Example 2__: A user wants to get expression values using case ids and gene ids.
+
+```Filter
+{
+  "case_ids": [
+    "6d4f38db-a97b-4dc0-8dc5-2ac7f2cc5e38",
+    "e3b32485-b204-43a7-93a5-601408fcdf96"
+  ],
+  "gene_ids": [
+    "ENSG00000141510",
+    "ENSG00000181143"
+  ],
+  "unit": "median-centered"
+}
+```
+
+```Shell
+curl -X 'POST' \
+  'https://api.gdc.cancer.gov/gene_expression/values' \
+  -H 'accept: text/tab-separated-values' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "case_ids": [
+    "6d4f38db-a97b-4dc0-8dc5-2ac7f2cc5e38",
+    "e3b32485-b204-43a7-93a5-601408fcdf96"
+  ],
+  "gene_ids": [
+    "ENSG00000141510",
+    "ENSG00000181143"
+  ],
+  "unit": "median-centered"
+}'
+```
+
+```Response
+gene_id	e3b32485-b204-43a7-93a5-601408fcdf96
+ENSG00000141510	6.93200
+ENSG00000181143	10.77660
+```
+
+### Gene Expression Gene Selection Endpoint
+
+Select the most variably expressed genes for a collection of cases and collection of genes.
+
+The request must define a collection of cases, a collection of genes, and a selection size. A minimum expression value may optionally be defined.
+
+A collection of cases must be defined by exactly one of the following:
+* `case_set_id`
+* `case_ids`
+* `cohort_id`
+* `case_filters`
+
+A collection of genes must be defined by exactly one of the following:
+* `gene_set_id`
+* `gene_ids`
+* `gene_type`
+
+A selection size (`selection_size`) defines the maximum number of genes to select.
+
+An optional threshold (`min_median_log2_uqfpkm`) defines a minimum value for expression. Defaults to `1`.
+
+__Example 1__: A users wants to retrieve the most variably expressed gened for a given cohort id and all protein-coding genes.
```

Comment:
s/users/user

Reply:
s/gened/genes

Reply:
deleted this example because it contained cohort_id and case_filters